prov/efa/src/rdm/efa_rdm_msg.c

```diff
@@ -158,7 +159,13 @@ ssize_t efa_rdm_msg_post_rtm(struct efa_rdm_ep *ep, struct efa_rdm_ope *txe, int
 	if (!efa_rdm_pkt_type_is_supported_by_peer(rtm_type, peer))
 		return -FI_EOPNOTSUPP;
 
-	return efa_rdm_ope_post_send(txe, rtm_type);
+	err = efa_rdm_ope_post_send(txe, rtm_type);
```

Comment:
so `efa_rdm_ope_post_send_handle_error` currently handles the sender side registration failure? Better to have a comment here.

Reply:
Yes that's correct. What exactly do you want as the comment? I have a comment inside the `efa_rdm_ope_post_send_handle_error` function

Reply:
that is fine for now